code/game/machinery/vending/vendor_types/antag/antag_predator.dm

```diff
@@ -220,7 +234,7 @@ GLOBAL_LIST_INIT(cm_vending_blooded_thrall, list(
 	desc = "A gear rack for hunting."
 	icon = 'icons/obj/items/hunter/pred_vendor.dmi'
 	icon_state = "pred_vendor_elder_left"
-	req_access = list(ACCESS_YAUTJA_ELITE, ACCESS_YAUTJA_ELDER, ACCESS_YAUTJA_ANCIENT)
+	req_access = list(ACCESS_YAUTJA_ELITE)
```

Comment:
as I explained in discord there's a separate variable you can use to make it so that you only need one of the listed accesses rather than just assuming higher ranks always have the access of lower ranks